app/CineHub/app/src/main/java/org/cinehub/javabeans/MovieRoom.java

```diff
@@ -0,0 +1,33 @@
+package org.cinehub.javabeans;
```

Comment:
This javabeans are meant to be merged with the model classes from the API or they should be independent?

Reply:
They are meant as placeholders, with the arrival of #28 to our main branch they'll be deprecated.